webapp/app/protected/controller.js

```diff
@@ -7,6 +7,13 @@ export default Ember.Controller.extend({
   name: config.APP.name,
   version: config.APP.version,
 
+  isNotAdmin: Ember.computed('session.user', 'session.user', function() {
```

Comment:
This function should be in the user model so that it could be accessed this way :
this.session.user.isAdmin